validator/test/fixtures/gke_enable_alias_ip_ranges/constraints/enable_alias_ip_ranges/data.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: constraints.gatekeeper.sh/v1alpha1
+kind: GCPGKEEnableAliasIPRangesConstraintV1
+metadata:
+  name: enable_alias_ip_ranges
+  annotations:
+    description: Ensure Kubernetes Cluster is created with Alias IP ranges enabled
+    # This constraint is not certified by CIS.
+    bundles.validator.forsetisecurity.org/cis-v1.1: 7.13
```

Comment:
Just to be consistent, can you remove the annotation from test data?

Reply:
@joecheuk  sure, done in latest push